main.cpp

```diff
@@ -79,43 +87,84 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+bool allowedToDrinkAlcohol (bool adult, bool insideBar, bool openHours)
+{
```

Comment:
- [ ] revise, pick a function name that represents an action and contains a verb